contracts/token/ERC721/ERC721.sol

```diff
@@ -148,7 +149,7 @@ contract ERC721 is ERC165, IERC721 {
      * @param tokenId uint256 ID of the token to be transferred
      */
     function safeTransferFrom(address from, address to, uint256 tokenId) public {
-        safeTransferFrom(from, to, tokenId, "");
+        safeTransferFrom(from, to, tokenId);
```

Comment:
What is the reason behind this change? This makes it be an infinite loop.

Reply:
Sorry, I forgot to remove this.